.drone.star

```diff
@@ -285,6 +291,7 @@ def gui_tests(squish_parameters = "", server_type = "oc10"):
             "SQUISH_PARAMETERS": squish_parameters,
             "STACKTRACE_FILE": "%s/stacktrace.log" % dir["guiTestReport"],
             "PLAYWRIGHT_BROWSERS_PATH": "%s/.playwright" % dir["base"],
+            "OWNCLOUD_CORE_DUMP": 1,
```

Comment:
@TheOneRing Can I use `OWNCLOUD_CORE_DUMP` as env like this?

Reply:
Yes it is an env var and this should work.

Reply:
The value seems alright, even though env vars are usually strings. As long as it runs (I assume Python just implicitly converts this to `str` where necessary) this is fine.

Reply:
yeah, env values will be treated as strings